src/index.ts

```diff
@@ -49,14 +49,16 @@ export function badgen ({
 
   label = sanitize(label)
   status = sanitize(status)
+  const accessibleText = createAccessibleText({label, status})
 
   if (style === 'flat') {
-    return `<svg width="${scale * width / 10}" height="${scale * 20}" viewBox="0 0 ${width} 200" xmlns="http://www.w3.org/2000/svg"${xlink}>
+    return `<svg width="${scale * width / 10}" height="${scale * 20}" viewBox="0 0 ${width} 200" xmlns="http://www.w3.org/2000/svg"${xlink} role="img" aria-label="${sanitizeAttribute(accessibleText)}">
+  <title>${accessibleText}</title>
```

Comment:
**Added:**
- `aria-label`
- `<title>`
- `role="img"`

The average screen reader experience should now be:

> ${Label}: ${status}, graphic